classes/submission/reviewAssignment/ReviewAssignment.php

```diff
@@ -682,12 +685,18 @@ public function getStatus()
             }
         } elseif ($this->getDateAcknowledged()) { // reviewer thanked...
             if ($this->getConsidered() == self::REVIEW_ASSIGNMENT_UNCONSIDERED) { // ...but review later unconsidered
-                return self::REVIEW_ASSIGNMENT_STATUS_RECEIVED;
+                return self::REVIEW_ASSIGNMENT_STATUS_VIEWED;
             }
             return self::REVIEW_ASSIGNMENT_STATUS_THANKED;
         } elseif ($this->getDateCompleted()) { // review submitted...
-            if ($this->getConsidered() != self::REVIEW_ASSIGNMENT_UNCONSIDERED && $this->isRead()) { // ...and confirmed by an editor
-                return self::REVIEW_ASSIGNMENT_STATUS_COMPLETE;
+            if ($this->getConsidered() != self::REVIEW_ASSIGNMENT_UNCONSIDERED) {
+                // if it was read and confirmed by the editor
+                if ($this->isRead()) {
+                    return self::REVIEW_ASSIGNMENT_STATUS_COMPLETE;
+                    // If it was viewed by the editor but not confirmed
+                } else {
+                    return self::REVIEW_ASSIGNMENT_STATUS_VIEWED;
+                }
```

Comment:
I would expect to return REVIEW_ASSIGNMENT_STATUS_VIEWED, when $this->getConsidered() === REVIEW_ASSIGNMENT_VIEWED.

But it has been long day...

Reply:
The logic here is pretty the same but I can narrow the results with `else if` instead of `else` just in case.